.gitignore

```diff
@@ -1 +1,5 @@
-node_modules/
\ No newline at end of file
+node_modules/
+# Ignore compiled files
+src/**/*.js
```

Comment:
that would indeed work, but it would require `tsc` to be installed so we need that as an additional dependency... I wouldn't do at this point in time. We can have a discussion about this when the project grows :-) thinking about it again, simply automating the publish-to-npm part would propably be the preferable solution